VikingQuiz.UI/src/Components/FormComponent/FormInput/FormInput.tsx

```diff
@@ -2,10 +2,14 @@ import * as React from 'react';
 import './FormInput.css';
 
 const formInput: any = (props: any) => {
+    let className = 'myinput form-control';
+    if (props.ErrorMessage !== ""){
```

Comment:
if u want to check that values in not empty/has some value u can use if (props.ErrorMessage) dont need any '!=='

cleaner version will be better
`const formInput: any = (props: any) => {
    const defaultInputClass = 'myinput form-control';
    const errorInputClass = props.ErrorMessage ? ' my-error' : '';
    return (
       <div className="formgroup col-xs-12">
        <label className="formlabel" htmlFor={props.InputId}>{props.InputLabel}</label>
        <input id={props.InputId} name={props.name} className={defaultInputClass + errorInputClass} type={props.InputType} onChange={props.changed}/>
        <div className="formerror">{props.ErrorMessage}</div>
      </div>
    )
}`

Reply:
try to name variables what they are doing, and to assign them somewhere that is easier to be visible and modified in future